devauth/devauth.go

```diff
@@ -1213,31 +1213,24 @@ func verifyTenantClaim(ctx context.Context, verifyTenant bool, tenant string) er
 }
 
 func (d *DevAuth) VerifyToken(ctx context.Context, raw string) error {
-
 	l := log.FromContext(ctx)
 
 	token := &jwt.Token{}
-
 	err := token.UnmarshalJWT([]byte(raw), d.jwt.FromJWT)
```

Comment:
I thought this is the place where signature is being validated

Reply:
I don't understand your comment. The signature verification is implemented in FromJWT, which I'm changing with this PR. It now uses `parser.ParseUnverified`, which doesn't perform the verification. I implemented a new method called `Validate` which performs the signature verification and it is called at a later point in time in the `VerifyToken` function.

Reply:
right; thanks for the clarification and sorry for the noise